holopy/scattering/scatterer/scatterer.py

```diff
@@ -99,7 +99,7 @@ def index_at(self, points, background=0):
         index = np.ones_like(domains, dtype=dtype) * background
         for i, n in enumerate(ns):
             index[domains==i+1] = n
-        return index
+        return xr.DataArray(index, attrs = {'_image_scaling':[0,self.n.max()], 'voxel':1})
```

Comment:
The `self.n.max()` call is tripping up a few unittests on travis. Specifically, it looks like n is sometimes defined as a float, list, or tuple in tests so the max call is failing. Wrapping `self.n` in `hp.core.metadata.ensure_array` should solve.